purchases/src/main/kotlin/com/revenuecat/purchases/models/GooglePurchasingData.kt

```diff
@@ -17,6 +17,7 @@ sealed class GooglePurchasingData : PurchasingData {
         val optionId: String,
         val productDetails: ProductDetails,
         val token: String,
+        val billingPeriod: Period?,
```

Comment:
I don't love that this is an optional, but I made it optional to gracefully handle converting `GoogleSubscriptionOption`s to `GooglePurchasingData.Subscription`. When we do that, `GoogleSubscriptionOption` doesn't have a single billing period, but instead has a list of `PricingPhase`s. It's technically possible for that list to be empty, and when so, we set the `GooglePurchasingData.billingPeriod` to null.

This shouldn't happen, but I wanted to avoid throwing if possible. Let me know if you can think of a better way to handle this!

Reply:
As you said, this should never happen since there should always be at least one pricing phase... We might want to add a log in this case, but yeah, I think it's ok to keep nullable 👍

Reply:
Sounds good, I'll add in a log 👍